.gitignore

```diff
@@ -1 +1,4 @@
 gin-swagger
+
+.idea
```

Comment:
This should be in your global `gitignore` :)

Reply:
Never used a global `gitignore` before. But can start today with one.